src/test/java/org/openrewrite/staticanalysis/DefaultComesLastTest.java

```diff
@@ -264,4 +264,39 @@ boolean foo() {
           )
         );
     }
+
+    @Test
+    void defaultComesLastCreateDuplicate() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              class Test {
+                  int n;
+                  boolean foo() {
+                      switch (n) {
+                          default:
+                          case DERIVATION_MODE:
+                            System.out.println("default");
+                            break;
+                      }
+                  }
+              }
+              """,
+            """
+              class Test {
+                  int n;
+                  boolean foo() {
+                      switch (n) {
+                          case DERIVATION_MODE:
+                            System.out.println("default");
+                            break;
+                          default:
```

Comment:
Perhaps it's too early for me, but shouldn't the behavior preserving change due to fall through be this instead?
```suggestion
                          case DERIVATION_MODE:
                          default:
                            System.out.println("default");
                            break;
```

Reply:
So you are suggesting the statements and break from case should move to default?, I am trying to understand the actual response of this. The statements & break belongs to case should be always part of that case right? It would be great if you can provide use/test case of this

Reply:
@nbenjamin whatever the outcome, the recipe should not change the behaviour. In your test the default case no longer prints to stdout. The DERIVATION_MODE case should rather fall through to the default case, which holds the action.

Reply:
Indeed what @delanym also says: we should not change the behavior; only put the default case last. In this case the test you had added was asserting a behavior change; I've just pushed up my suggestion that asserts what I think should be the right replacement here, and we'd need to get the recipe implementation to line up.